analyzers/src/SonarAnalyzer.VisualBasic/Rules/TooManyLabelsInSwitch.cs

```diff
@@ -18,31 +18,39 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-namespace SonarAnalyzer.Rules.VisualBasic
+namespace SonarAnalyzer.Rules.VisualBasic;
+
+[DiagnosticAnalyzer(LanguageNames.VisualBasic)]
+public sealed class TooManyLabelsInSwitch : TooManyLabelsInSwitchBase<SyntaxKind, SelectStatementSyntax>
 {
-    [DiagnosticAnalyzer(LanguageNames.VisualBasic)]
-    public sealed class TooManyLabelsInSwitch : TooManyLabelsInSwitchBase<SyntaxKind, SelectStatementSyntax>
-    {
-        protected override DiagnosticDescriptor Rule { get; } =
-            DescriptorFactory.Create(DiagnosticId, MessageFormat,
-                isEnabledByDefault: false);
+    private static readonly ISet<SyntaxKind> IgnoredStatementsInSwitch = new HashSet<SyntaxKind> { SyntaxKind.ReturnStatement, SyntaxKind.ThrowStatement };
+
+    protected override DiagnosticDescriptor Rule { get; } =
+        DescriptorFactory.Create(DiagnosticId, string.Format(MessageFormat, "Select Case", "Case"),
```

Comment:
```suggestion
        DescriptorFactory.Create(DiagnosticId, string.Format(MessageFormat, "Select", "Case"),
```
Isn't it much less confusing this way?

Reply:
I don't know what the common usage of `Select` in VB.NET.
I kept what was used in the previous implementation.

The `Case` in `Select Case` seems to be optional. Personally, I would keep the `Case`.